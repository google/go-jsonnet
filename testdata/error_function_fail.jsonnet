error (function(x) 42)