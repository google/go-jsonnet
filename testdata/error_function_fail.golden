RUNTIME ERROR: Couldn't manifest function in JSON output.
-------------------------------------------------
	testdata/error_function_fail:1:1-23	$

error (function(x) 42)

-------------------------------------------------
	During evaluation	


